### DISCUSSION TITLE
Registration for the 2024 RDKit UGM now open

### DISCUSSION THREAD

**Discussion by greglandrum** [rdkit/rdkit#7243] [General]
The (free) registration for the 2024 RDKit UGM, being held from 11-13 September at the ETH in Zurich, Switzerland, is now open:
https://www.eventbrite.com/e/860637719587

You can submit proposals to do talks, tutorials, lightning talks, and posters here:
https://forms.gle/5GK5ej7hCdPguwKz8

As in the past couple of years, we will stream the talks for people who cannot attend in person.

I'm looking forward to seeing many of you in Zurich!

**Comment by UnixJunkie** [comment]
Is presenting from remote possible?
I would _love_ to come in person, but it _is_ quite expensive to come from where I live and my employer might not pay for this...

**Reply by greglandrum** [reply]
Hi @UnixJunkie, remote presentation is not possible. I know this makes it difficult for people who are far away to present, but our experience is that remote presentations are somewhat disruptive.

**Comment by fbaensch-beilstein** [comment]
Hi @greglandrum,
Is there a deadline for poster submissions?

**Comment by drizhina** [comment]
Hi @greglandrum,
Is there a deadline for talk proposal submission?

**Reply by greglandrum** [reply]
@drizhina we are wrapping up the agenda now, so today or early tomorrow would be good
